contents/handbook/company/branding.md

```diff
@@ -200,7 +200,7 @@ This uses 'Extra Large Text', as defined in the 'Fonts' section.
 
 Underneath these 5 categories are the sub sections, which use 'Normal Text', as defined in the 'Fonts' section.
 
-Side Menus, found on pages such as Docs, are to be a Light Grey (#F0F0F0) and 430px wide. The text and drop down options should be fixed so that even when reading the consumer can still have quick access to other areas within the site. 
```

Comment:
The word "dropdown" is generally more commonly used than "drop down", and the word "dropdown" also appears in this same document on line 207. Editing this instance for consistency.